common/religion/religions/arcane_group.txt

```diff
@@ -239,7 +240,7 @@
 			holy_site = darnassus
 
 			doctrine = tenet_aniconism
-			doctrine = tenet_communal_identity
+			doctrine = tenet_hedonistic
```

Comment:
Actually, since this religion is pretty much just elves now. You can Proud virtue here.
<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46384992/114222821-1c9f5600-9980-11eb-8a29-7edd2ed000bb.png)

</details>